lib/MusicBrainz/Server/Form/Filter/Work.pm

```diff
@@ -31,6 +41,15 @@ sub options_role_type {
     ];
 }
 
+sub options_language_id {
+    my ($self, $field) = @_;
+    return [
+        { value => '-1', label => lp('[none]', 'language') },
```

Comment:
Hmm. This would be inconsistent with all other filters - do you think we should change all of them to `[not set]`? That might work too.

Reply:
It doesn’t seem necessary to change it for all the other filters as long as they don’t have a value that can be mistaken with.

Reply:
I'm not happy with the inconsistency - I'd say either all should be `[none]` or all `[not set]` if all do the same thing. I'm okay with either of the two options though.

Reply:
> I'm not happy with the inconsistency

It would be at most a small but justified difference in the label used to say that the filtered items have none of the other options. Each filter has different values anyway. Some filters don’t even have an option `[none]` at all. There are so many inherent differences between filters that it doesn’t seem useful to have the exact same string for different cases. It would also require changing 6 already translated strings.

> I'm okay with either of the two options though.

That is a real _inconsistency_ as in a contradiction with discussing how to find a better replacement for `[none]` which isn’t okay for lyrics languages.

Reply:
> That is a real inconsistency as in a contradiction with discussing how to find a better replacement for [none] which isn’t okay for lyrics languages.

Well, I thought you said "especially if no lyrics is missing" and now that's fixed, but whatever, not worth arguing I guess. If you really feel that strongly about this one, I'll make it inconsistent.

Reply:
> "especially if no lyrics is missing" and now that's fixed

It wasn’t related to the other comment. There will always be cases where artists have no works with no lyrics.

For example: http://localhost:5000/artist/25848dee-8562-4a78-b375-3a80b61da629/works

Reply:
I thought in that case it would be obvious after filtering what it meant :man_shrugging: But I made the change, anyway.